src/main/java/com/github/vkorobkov/jfixtures/config/structure/tables/Tables.java

```diff
@@ -0,0 +1,46 @@
+package com.github.vkorobkov.jfixtures.config.structure.tables;
+
+import com.github.vkorobkov.jfixtures.config.structure.Section;
+import com.github.vkorobkov.jfixtures.config.structure.util.TableMatcher;
+import com.github.vkorobkov.jfixtures.config.yaml.Node;
+
+import java.util.stream.Stream;
+
+public class Tables extends Section {
+    private static final String SECTION_APPLIES_TO = "applies_to";
+    private static final String SECTION_PRIMARY_KEY = "pk";
+    private static final String SECTION_GENERATE = "generate";
+    private static final String SECTION_COLUMN = "column";
+    private static final String PK_DEFAULT_COLUMN_NAME = "id";
+
+    private final String name;
+
+    public Tables(Node node, String name) {
+        super(node);
+        this.name = name;
+    }
+
+    public boolean shouldAutoGeneratePk() {
```

Comment:
`shouldAutoGeneratePk` and `getCustomColumnForPk` have many shared code. 
let's do a common method like
```java
T Optional<T> getProperty(String ... sections)
```
?